internationalize GPS error popup message Fix

Joseph-M-Newman opened the pull request:
# Description
GPS error popup now displays in proper languages, English, Spanish, French.
Beforehand, the alert message had been hardcoded to only English. 

Fixes #705 
Imported translate component. Added three new data.js JSON lines that were included in each of the following: en, fr, es. Detailed to the correct error messages for the GPS latitude and longitude coordinates being outside the range (-90 -> 90) & (-180 -> 180). This will fix the Issue 

## Type of change

- [ ] Note merging this changes the node modules
- [ ] Note merging this changes the database configuration.
- [ ] This change requires a documentation update

## Checklist

- [x] I have followed the [OED pull request](https://openenergydashboard.github.io/developer/pr.html) ideas
- [x] I have removed text in ( ) from the issue request

## Limitations

## Addition Addons
This fix is located src/client/app/utils/calibration.ts in function isValidGPSInput.

<img width="446" alt="Screen Shot 2022-02-16 at 3 47 40 PM" src="https://user-images.githubusercontent.com/17994162/154376685-c31fdfe7-e653-4bcf-b1cf-49060feff50f.png">

Joseph-M-Newman committed:
internationalize GPS error popup message #705

GPS error popup now displays in proper languages, English, Spanish, French

huss commented on src/client/app/utils/calibration.ts:
```diff
@@ -118,9 +119,11 @@ export function isValidGPSInput(input: string): boolean {
 	const result = latitudeConstraint && longitudeConstraint;
 	if (!result) {
-		window.alert('invalid gps coordinate, ' +
+		/** window.alert('invalid gps coordinate, ' +
```
Please remove the old code as it is no longer needed. It might be good to then add a comment saying something similar to:
// GPS values not in valid range
Since the internationalized string is not visible so it is harder to know what is being tested.
Note one of the checks failed because of this comment since it was /** so jsdoc but not really intended for this. I assume this failure will go away with this edit but we need to be sure all the checks pass after it gets past that.

huss commented on src/client/app/translations/data.js:
```diff
@@ -139,6 +139,7 @@ const localeData = {
 		"insufficient.readings": "Insufficient readings data to process comparison for ",
 		"invalid.email.password": "Invalid email/password combination",
 		"invalid.number": "Please submit a valid number (between 0 and 2.0)",
+		"intlError": "Invalid GPS coordinate, latitude must be an integer between -90 and 90, longitude must be an integer between -180 and 180",
```
I think we should change the name of the string to be clearer about what it is for. Right above is input issues for GPS so I propose intlError -> input.gps.range. The place in the file needs to be moved so alphabetical with this change.

Joseph-M-Newman replied:
Sure thing, I'll make both these changes right now and send it back to you.

huss reviewed (changes requested):
I thank the team for its first PR. It works as expected so the work is fine. I've asked for a couple of changes before it is approved/merged.

Joseph-M-Newman committed:
Fix#705

Removed Old code, and formated Data.js a Little better. Changed intlError to input.gps.range

huss committed:
fix alphabetical order

huss reviewed (approved):
Looks good. I pushed a small change to get the alphabetical order correct for the translation file. Testing indicates it works fine and is ready to merge. Congratulations to the team for their first PR merge!
